src/DetectorConstruction.cc

```diff
@@ -344,7 +362,7 @@ G4VPhysicalVolume* DetectorConstruction::Construct()
   G4double particleGuideAlphaLowerEndcap = 0*deg;
   G4double particleGuideAlphaUpperEndcap = 0*deg;
   G4double particleGuideWallSize = 3*mm;
-  G4ThreeVector particleGuideShift(0.*cm, -25*cm, 33.5*cm);
+  G4ThreeVector particleGuideShift(0.*cm, -25*cm + 0.5*cm, 33.5*cm);
```

Comment:
maybe use the thickness you defined previously in the definition of the Shifts and particleGuideZHalfLength and not the raw value